pkg/cache/cache_test.go

```diff
@@ -111,30 +111,22 @@ func TestCacheClusterQueueOperations(t *testing.T) {
 					AllocatableResourceGeneration: 1,
 					NamespaceSelector:             labels.Nothing(),
 					FlavorFungibility:             defaultFlavorFungibility,
-					Usage: resources.FlavorResourceQuantities{
-						{Flavor: "default", Resource: corev1.ResourceCPU}: 0,
-					},
-					AdmittedUsage: resources.FlavorResourceQuantities{
-						{Flavor: "default", Resource: corev1.ResourceCPU}: 0,
-					},
-					Status:     active,
-					Preemption: defaultPreemption,
-					FairWeight: oneQuantity,
+					Usage:                         resources.FlavorResourceQuantities{},
```

Comment:
you can use `cmpopts.EquateEmpty` to avoid having to define these empty maps.

Reply:
Done.